### PR TITLE
Fixed Chunk Problems

### PR DESCRIPTION
**Pull request by SuperXingKong**
### Purpose and Effect of This Pull Request

This pr may fix some issue about Chunk. And I am not sure whether it is ok or not. But at least it is stable.

### Testing

- [x] Working well.

### CODE REVIEW

File: src/pocketmine/Player.php [hunk 1]
```diff
@@ -4321,14 +4306,14 @@ public function attack($damage, EntityDamageEvent $source){
...
-		$pitch = $pitch === null ? $this->pitch : $pitch;
+	public function sendPosition(Vector3 $pos, float $yaw = null, float $pitch = null, int $mode = MovePlayerPacket::MODE_NORMAL, array $targets = null, float $baseOffsetOverride = null){
+		$yaw = $yaw ?? $this->yaw;
+		$pitch = $pitch ?? $this->pitch;
 
 		$pk = new MovePlayerPacket();
 		$pk->eid = $this->getId();
 		$pk->x = $pos->x;
-		$pk->y = $pos->y + $this->getEyeHeight();
+		$pk->y = $pos->y + ($baseOffsetOverride ?? $this->baseOffset);
```
Comment (NycuRO) [inline src/pocketmine/Player.php]: not need ( )

File: src/pocketmine/entity/Entity.php [hunk 2]
```diff
@@ -2138,7 +2138,7 @@ public function teleport(Vector3 $pos, $yaw = null, $pitch = null){
 		$pos = $ev->getTo();
 
 		$this->setMotion($this->temporalVector->setComponents(0, 0, 0));
-		if($this->setPositionAndRotation($pos, $yaw === null ? $this->yaw : $yaw, $pitch === null ? $this->pitch : $pitch) !== false){
+		if($this->setPositionAndRotation($pos, $yaw ?? $this->yaw, $pitch ?? $this->pitch) !== false){
```
Comment (NycuRO) [inline src/pocketmine/entity/Entity.php]: can you define $yaw , $pitch null?

Comment (Leo3418) [inline src/pocketmine/entity/Entity.php]: @NycuRO I recommend a reading for you https://stackoverflow.com/questions/23752980/php-meaning-of-question-mark-colon-operator

Comment (NycuRO) [inline src/pocketmine/entity/Entity.php]: Ohh, thanks @Leo3418

**Comment by SuperXingKong** [comment]
@andris155 I didn't change any APIs...

**Pull request merged**